### PR TITLE
Fix default value for imported packages

### LINKED ISSUE #1744: Default value for package import not found
**Describe the bug**
The default value `sklearn.cluster.MiniBatchKMeans` specified in the `/python/imports/` endpoint is not found:
```
{
  "error": "No package name records for package 'sklearn.cluster.MiniBatchKMeans' in database",
  "parameters": {
    "import_name": "sklearn.cluster.MiniBatchKMeans"
  }
}
```
Environment: production.

**To Reproduce**
Go to https://khemenu.thoth-station.ninja/api/v1/ui/#/PythonPackages/get_package_from_imported_packages and query imports for the default value `sklearn.cluster.MiniBatchKMeans` of `import_name`.

**Expected behavior**
Default value is present in the database.

### PR DESCRIPTION
**Pull request by mayaCostantini**
## Related Issues and Dependencies

Fixes #1744 

## This introduces a breaking change

- No

## This should yield a new module release

- No

### CODE REVIEW

**Review by harshad16 — CHANGES_REQUESTED** [review]
Ah interesting, 
let's check what could be the issue with the existing example.

File: openapi/openapi.yaml [hunk 1]
```diff
@@ -1529,7 +1529,7 @@ components:
         Names of the Python Packages (name, version, index) for the given import package name
       schema:
         type: string
-        example: sklearn.cluster.MiniBatchKMeans
+        example: sklearn
```
Comment (harshad16) [inline openapi/openapi.yaml:1532]: However, lets use the this
```suggestion
        example: sklearn.neural_network
```

Verified on Prod: https://khemenu.thoth-station.ninja/api/v1/python/imports?import_name=sklearn.neural_network

**Review by harshad16 — CHANGES_REQUESTED** [review]
previous comment some how missed my suggestion post

File: openapi/openapi.yaml (hunk 1 above)
Comment (mayaCostantini) [reply openapi/openapi.yaml:1532]: Thank you! Adding the change :+1:

**Review by harshad16 — APPROVED** [review]
/lgtm
/approve

good catch :100:

**Review by fridex — APPROVED** [review]
lgtm! Thanks! 👍🏻

**Pull request merged**